packages/ra-test/package.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "ra-test",
+    "version": "3.12.0",
+    "description": "Test utilities for react-admin, a frontend Framework for building admin applications on top of REST services, using ES6, React",
+    "files": [
+        "*.md",
+        "lib",
+        "esm",
+        "src"
+    ],
+    "main": "lib/index",
+    "module": "esm/index.js",
+    "types": "esm/index.d.ts",
+    "sideEffects": false,
+    "authors": [
+        "François Zaninotto",
+        "Gildas Garcia"
+    ],
+    "repository": "marmelab/react-admin",
+    "homepage": "https://github.com/marmelab/react-admin#readme",
+    "bugs": "https://github.com/marmelab/react-admin/issues",
+    "license": "MIT",
+    "scripts": {
+        "build": "yarn run build-cjs && yarn run build-esm",
+        "build-cjs": "rimraf ./lib && tsc",
+        "build-esm": "rimraf ./esm && tsc --outDir esm --module es2015",
+        "watch": "tsc --outDir esm --module es2015 --watch"
+    },
+    "devDependencies": {
+        "@redux-saga/testing-utils": "^1.0.2",
+        "@types/history": "^4.7.2",
+        "@types/node-polyglot": "^0.4.31",
+        "@types/query-string": "5.1.0",
+        "@types/react-redux": "^7.1.1",
+        "@types/react-router": "^5.1.0",
+        "@types/react-router-dom": "^5.1.0",
+        "connected-react-router": "^6.5.2",
+        "cross-env": "^5.2.0",
+        "final-form": "^4.20.0",
+        "history": "^4.7.2",
+        "ignore-styles": "~5.0.1",
+        "ra-core": "^3.12.0",
```

Comment:
I think this should be a peerDependency, to avoid circular dependencies.

Reply:
It's in both dev and peer dependencies